web/src/helpers/zonedata.js

```diff
@@ -34,18 +34,10 @@ export function getProductionCo2Intensity(mode, zoneData) {
     );
 }
 
-export function getExchangeCo2Intensity(mode, zoneData, electricityMixMode) {
-  const exchange = (zoneData.exchange || {})[mode];
-  const exchangeCo2Intensity = (zoneData.exchangeCo2Intensities || {})[mode];
+export function getExchangeCo2Intensity(key, zoneData) {
+  const exchangeCo2Intensity = (zoneData.exchangeCo2Intensities || {})[key];
 
-  return exchange > 0
-    ? (
-      exchangeCo2Intensity
-    ) : (
-      electricityMixMode === 'consumption'
-        ? zoneData.co2intensity
-        : zoneData.co2intensityProduction
-    );
+  return exchangeCo2Intensity;
```

Comment:
I'm happy for this simplification! :+1: 

I didn't quite understand this code when I was refactoring so I decided not to touch this logic before :smile: